tools/android_lint/bin/main.dart

```diff
@@ -73,7 +73,7 @@ Future<int> runLint(ArgParser argParser, ArgResults argResults) async {
 <!-- WILL AUTOMATICALLY FIND ALL .java FILES AND INCLUDE THEM HERE       -->
 <project>
   <sdk dir="${androidSdkDir.path}" />
-  <module name="FlutterEngine" android="true" library="true" compile-sdk-version="android-T">
+  <module name="FlutterEngine" android="true" library="true" compile-sdk-version="android-U">
```

Comment:
`android-U` is a guess here. I'm assuming the format is just first letter of the version name, but I can't find documentation